src/services/dbgap_telemetry_report.py

```diff
@@ -0,0 +1,49 @@
+import requests
+import xmltodict
+
+class SampleNotFoundError(Exception):
+    """Exception raised when a sample is not found."""
+    def __init__(self, alias):
+        self.alias = alias
+        super().__init__(f"Sample with alias '{alias}' not found")
+
+class DbgapTelemetryWrapper:
+    def __init__(self, phs_id=None):
+        self.endpoint = f"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/GetSampleStatus.cgi?rettype=xml&study_id={phs_id}"
+        self.phs_id = phs_id
+
+    def call_telemetry_report(self):
+        """Example xml - https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/GetSampleStatus.cgi?rettype=xml&study_id=phs000452"""
+        response = requests.get(
+            self.endpoint, 
+            headers={"Content-Type": "application/json"}
+        )
+        status_code = response.status_code
```

Comment:
Looks like the `status_code` isn't used anywhere - Do we want to check and see if the response status was a valid one before returning the response body?

Reply:
Oh yea absolutely. Good catch!